### PR TITLE
Add comments to Storage.cs functions.

### PR DESCRIPTION
**Pull request by hypixus**

### CODE REVIEW

**Comment by hypixus** [comment]
This just adds `<summary>` to each function in Storage.cs file.

**Comment by hypixus** [comment]
@Top-Cat Please expect more PRs like this one, one for each file.

File: UnofficialSteamAuthenticator/Storage.cs [hunk 1]
```diff
@@ -59,7 +59,9 @@ public static SteamGuardAccount GetSteamGuardAccount(ulong steamId)
 
             return response;
         }
-
+        ///<summary>
+        ///Recieve SteamGuardAccount object for current session.
```
Comment (setoy) [inline UnofficialSteamAuthenticator/Storage.cs:63]: It's **receive** and it's missing the initial spaces after the _///_

File: UnofficialSteamAuthenticator/Storage.cs [hunk 2]
```diff
@@ -146,7 +148,9 @@ public static SessionData GetSessionData()
 
             return response;
         }
-
+        ///<summary>
+        ///Save Session object data to phone's storage.
```
Comment (setoy) [inline UnofficialSteamAuthenticator/Storage.cs:152]: Missing spaces

File: UnofficialSteamAuthenticator/Storage.cs [hunk 3]
```diff
@@ -156,7 +160,9 @@ public static void PushStore(SessionData session)
 
             localSettings.Values["sessionJson"] = JsonConvert.SerializeObject(accounts);
         }
-
+        ///<summary>
+        ///Deletes account from phone's storage.
```
Comment (setoy) [inline UnofficialSteamAuthenticator/Storage.cs:164]: Missing spaces

File: UnofficialSteamAuthenticator/Storage.cs [hunk 4]
```diff
@@ -166,7 +172,9 @@ public static void Logout(ulong steamid)
 
             localSettings.Values["sessionJson"] = JsonConvert.SerializeObject(accounts);
         }
-
+        ///<summary>
+        ///Deletes current account from phone's storage.
```
Comment (setoy) [inline UnofficialSteamAuthenticator/Storage.cs:176]: Missing spaces

**Comment by Top-Cat** [comment]
param and returns tags would be nice too if you're trying to do this properly

**Comment by hypixus** [comment]
@setoy Thanks for review, I'll repair bugs in my code :D

**Pull request closed without merging**